indico_toolkit/indico_wrapper/dataset.py

```diff
@@ -139,9 +171,44 @@ def get_col_name_by_id(self, dataset_id: int, col_id: int) -> str:
         dataset = self.get_dataset(dataset_id)
         return next(c.name for c in dataset.datacolumns if c.id == col_id)
 
+    def upload_datafiles(self, filepaths: List) -> List[Dict]:
```

Comment:
I'd hide this method "_upload_datafiles"

Reply:
WIll do